two-tier-nodejsapp-migration-to-containers-on-Azure/README.md

```diff
@@ -0,0 +1,56 @@
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Ftwo-tier-nodejsapp-migration-to-containers-on-Azure%2Fazuredeploy.json)
+
+
+<!-- TOC -->
+
+1. [Solution Overview](#solution-overview)
+2. [Template Solution Architecture ](#template-solution-architecture)
+3. [Licenses and Costs ](#licenses-and-costs)
+4. [Prerequisites](#prerequisites)
+5. [Deployment Steps](#deployment-steps)
+6. [Deployment Time](#deployment-time)
+7. [Support](#support)
+
+
+<!-- /TOC -->
+
+## Solution Overview
+This Azure QuickStart template deploys the simulation infrastructure for on-premises two-tier Nodejs application and MongoDb on a single Ububtu server virtual machine.
+Once deployment finishes, you can connect to deployed VM and verify the Mongo database as well as Nodejs application and test application by launching the same in browser.
+
+## Template Solution Architecture
+This template will deploy:
+*	One storage account
+*	One Virtual Network with one subnets
+*	One Network Security Group
+*	One Public IP
+*	One Ubuntu Server VM
+*   Installation and configuration of Mongo database and Nodejs application
+
+
+<img src="images/onPremApp.jpg"/> 
+
+Post migration of the app and db using the guide, your Azure enviroment will be as per the below diagram
+<img src="images/ContainerApp.jpg"/> 
+## Licenses and Costs
+This uses Ubuntu Linux image which is freely available and no license is required to deploy the same. User will be charged hourly for Azure’s Linux VM rates. Click [here](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuServer?tab=PlansAndPrice) for pricing details.
+
+## Prerequisites
+*	Azure Subscription with specified payment method or Azure credits
+## Deployment Steps
+Build your Two-tier on-premises app simulation on Azure in a few simple steps:
+*	Launch the Template by click on Deploy to Azure button.
+*	Fill in all the required parameter values. Accept the terms and condition on click Purchase.
+*	Access the deployment job once it is successful. In deployment job output, you will find the Public IP Address of VMs which can be used connect to the VMs.
+*	Start the Nodejs (ToDo) application and launch the same in browser 
+*	Populate the application by entering some records 
+*	Open Mongo console and verify the database is successfully created.
+*	Follow the post deployment configuration document [here](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/two-tier-nodejsapp-migration-to-containers-on-Azure/Two-tier%20Nodejs%20App%20migration%20on%20Azure%20Container%20v0.1.pdf) to learn about further configuration.
+
+## Deployment Time
+The deployment takes about 20 minutes to complete.
+## Support
+For any support related questions, issues or customization requirements, please contact info@spektrasystems.com <br/>
+****End****
```

Comment:
remove the PDF file from the sample

Reply:
This template actually deploys the simulation of on-premises two-tier application. After that user need to follow the PDF guide to migrate the application to Azure PaaS offerings. (DB and Containers). So this PDF is required.

Reply:
Let's host it somewhere else and link to it - we can't track changes to a PDF

Reply:
Isit be OK if i make it as markdown file?

Reply:
deleted file and added reference to our git.